test_scripts/API/ATF_syncMsgVersion.lua

```diff
@@ -128,7 +128,7 @@ end
 
 function SDL_Core.getAPIFileVersion()
```

Comment:
This function looks unused, can likely remove. On another note it doesn't include patch version.